auth_backend/modules/superAdmin/apps.py

```diff
@@ -1,6 +1,5 @@
 from django.apps import AppConfig
 
-
-# all the admin stuff
+#all the admin stuff
 class AdminConfig(AppConfig):
-    name = 'modules.superAdmin'
+    name = 'apps.superAdmin'
```

Comment:
why is this changed

Reply:
it must be because of the merge. The modules folder was not renamed. Its fixed now.